packages/eslint-config-react-native-community/index.js

```diff
@@ -309,5 +309,11 @@ module.exports = {
     'jest/no-focused-tests': 1,
     'jest/no-identical-title': 1,
     'jest/valid-expect': 1,
+
+    // Prettier Plugin
+    'prettier/prettier': [
+      1,
+      {singleQuote: true, trailingComma: 'es5', semi: true},
```

Comment:
@iRoachie - I wonder how this section is related to the default content of `.prettierrc.js`: 

```
module.exports = {
  bracketSpacing: false,
  jsxBracketSameLine: true,
  singleQuote: true,
  trailingComma: 'all',
};
```

Does it override and/or make it (partially) obsolete?

Reply:
Not sure exactly, what i do know is that the contents of that file doesn’t get passed onto user projects and previously you had to have a .prettierrc of your own

Reply:
Okay thanks for your reply. If (part of?) that file is obsolete (after updating the [`@react-native-community/eslint-config` dependency](https://github.com/facebook/react-native/blob/master/template/package.json#L18)), maybe we should update the [template file](https://github.com/facebook/react-native/blob/master/template/_prettierrc.js) as well?

Reply:
@satya164 - Thanks for your comment! Could you give an example (or a link to one)?

Reply:
Prettier can format a wide-range of files such as markdown files (including code inside markdown files). The ESLint config only applies to TypeScript and JS files. Not sure what kind of example you're expecting?

Reply:
Okay never mind, I understand the reason. Thanks!